SCClassLibrary/Common/Control/Volume.sc

```diff
@@ -22,19 +22,20 @@ Volume {
 	}
 
 	sendSynthDef {
-
 		forkIfNeeded {
 			var synthNumChans = this.numChannels;
 			defName = (\volumeAmpControl ++ synthNumChans).asSymbol;
 			SynthDef(defName, { | volumeAmp = 1, volumeLag = 0.1, gate=1, bus |
-					XOut.ar(bus,
-						Linen.kr(gate, releaseTime: 0.05, doneAction:2),
-						In.ar(bus, synthNumChans) * Lag.kr(volumeAmp, volumeLag)
-					);
+				XOut.ar(bus,
+					Linen.kr(gate, releaseTime: 0.05, doneAction:2),
+					In.ar(bus, synthNumChans) * Lag.kr(volumeAmp, volumeLag)
+				);
 			}).send(server);
 
 			server.sync;
 
+			this.updateSynth;
+
 			if(updateFunc.isNil) {
 				ServerTree.add(updateFunc = {
```

Comment:
while we're at it... this use of an assignment as an expression makes me cringe, could you move it out to `updateFunc = { }; ServerTree.add(updateFunc)`? i'm warming up to brian's policy of cleaning up adjacent code when working on things.